### PR TITLE
Apply the release-notes-none label to cherrypicks

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Review by ericvn — DISMISSED** [review]

File: prow/cluster/cherrypicker_deployment.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ spec:
         - --dry-run=false
         - --use-prow-assignments=false
         - "--labels=cla: yes"
+        - --labels=release-notes-none
```
Comment (ericvn) [inline prow/cluster/cherrypicker_deployment.yaml:27]: Is the double --labels going to cause an issue?

Comment (ericvn) [reply prow/cluster/cherrypicker_deployment.yaml:27]: It looks like GitHub-endpoint has multiple.

Comment (jwendell) [reply prow/cluster/cherrypicker_deployment.yaml:27]: on the contrary. It only works this way. Comma separated will be interpreted as only one string.

Comment (jwendell) [reply prow/cluster/cherrypicker_deployment.yaml:27]: Played locally with https://github.com/kubernetes/test-infra/blob/master/prow/external-plugins/cherrypicker/main.go#L65 and https://github.com/kubernetes/test-infra/blob/master/prow/flagutil/strings.go

Comment (jwendell) [reply prow/cluster/cherrypicker_deployment.yaml:27]: Result of `fmt.Printf("%d - %#v\n", len(o.labels.Strings()), o.labels.Strings())`:
```
 ~/src/k8s/test-infra/prow/external-plugins/cherrypicker [master|✚ 1] 
12:23 $ go run main.go --labels=x1,x2,x3
1 - []string{"x1,x2,x3"}

✔ ~/src/k8s/test-infra/prow/external-plugins/cherrypicker [master|✚ 1] 
12:23 $ go run main.go --labels=x1 --labels=x2 --labels=x3
3 - []string{"x1", "x2", "x3"}
```

**Review by ericvn — APPROVED** [review]

File: prow/cluster/cherrypicker_deployment.yaml (hunk 1 above)
Comment (ericvn) [reply prow/cluster/cherrypicker_deployment.yaml:27]: Thanks

Comment (jwendell) [reply prow/cluster/cherrypicker_deployment.yaml:27]: > It looks like GitHub-endpoint has multiple.

yeah, they use the same backend to parse array of strings

Comment (brian-avery) [reply prow/cluster/cherrypicker_deployment.yaml:27]: Thanks.

**Review by brian-avery — APPROVED** [review]

**Review by howardjohn — APPROVED** [review]

File: prow/cluster/cherrypicker_deployment.yaml (hunk 1 above)
Comment (brian-avery) [inline prow/cluster/cherrypicker_deployment.yaml:27]: Any idea why the prior one is in quotes and this one is not?

Comment (brian-avery) [reply prow/cluster/cherrypicker_deployment.yaml:27]: I guess the space would change it.

Comment (jwendell) [reply prow/cluster/cherrypicker_deployment.yaml:27]: space. it could be the binary would interpret `yes` as another argument.

**Review by fejta — APPROVED** [review]

**Pull request merged**